ci/docs/build.sh

```diff
@@ -43,13 +43,13 @@ cd $PROJECT_WORKSPACE/docs
 make html
 
 #Commit to Website
-cd $DOCS_WORKSPACE
+cd "$DOCS_WORKSPACE"
 
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
 done
 
-mv $PROJECT_WORKSPACE/docs/_build/html/* $DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
+mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
```

Comment:
```suggestion
mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION"
```

looks like this is missing a closing `"`

Reply:
@jolorunyomi, I still don't see a closing `"` here.